ui/components/app/whats-new-popup/whats-new-popup.js

```diff
@@ -202,7 +202,7 @@ const renderFirstNotification = ({
       </div>
       {placeImageBelowDescription && imageComponent}
       {actionText && (
-        <Button
+        <ButtonPrimary
```

Comment:
Looks good ! just a question, what is the difference between `ButtonPrimary` and `Button` component?

Reply:
Button is essentially a switch statement that renders the different variants: primary, secondary, link. We initially thought it might be a good idea to build out the variants a separate components to prevent component bloat. Using either is fine. Using the individual variants may have some very minor performance benefits because it's not wrapped in an extra function